tasks/sep/027-holocene-system-config-upgrade-and-init-multi-chain/README.md

```diff
@@ -0,0 +1,30 @@
+# Holocene Hardfork Upgrade - `SystemConfig`
+
+Status: DRAFT, NOT READY TO SIGN
```

Comment:
what do we need for this to become ready to sign?
```suggestion
Status: DRAFT, NOT READY TO SIGN
```

Reply:
I updated it now, there are no blockers other than review of this PR.